### ISSUE TITLE
[A11y] Desktop/Mobile Explore Collection - Grid: Orange "All" tab and Active pagination

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1788]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Explore Collections

### Admin user?

- [ ] Yes

### Issue Description

Explore Collection - Grid: https://staging.oregondigital.org/explore_collections/all?locale=en&view=masonry

Failed Instance:
color-contrast: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

Path: .active > .stacker-container > .btn-outline.btn
Snippet: 
`<a class="btn btn-outline" href="/explore_collections/all?locale=en">All</a>`

### Suggested Fix

Fix the following:Element has insufficient color contrast of 2.65 (foreground color: #ffffff, background color: #e28a25, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

POSM Review for color options, these color combos pass the color contrast checker:

Orange background and Black Text
![orange_black](https://user-images.githubusercontent.com/8353860/128075882-6ee64d92-bcbe-44f6-af53-7b14837c8016.jpg)

Tab Example
 ![orange_black_tab](https://user-images.githubusercontent.com/8353860/128072463-cb0b238c-5469-43a0-9a92-14098eda3cc3.jpg)

Blue background and White Text
<img width="476" alt="blue_white" src="https://user-images.githubusercontent.com/8353860/128072249-d1c8d4d6-cae3-4964-9bd5-ad866f35eb81.png">

Tab Example
<img width="740" alt="blue_white_tab" src="https://user-images.githubusercontent.com/8353860/128073016-72759901-d96a-47d9-89cc-ad95f367e631.png">

Orange and Blue Text, using accessible color generator
AA Compliant
<img width="603" alt="Screen Shot 2021-08-10 at 1 57 14 PM" src="https://user-images.githubusercontent.com/8353860/128934116-47a33941-1f1b-48a6-b171-a25d7df5906a.png">

AAA Compliant
<img width="610" alt="Screen Shot 2021-08-10 at 1 58 43 PM" src="https://user-images.githubusercontent.com/8353860/128934170-0919cc97-3524-4756-9413-e3a134f197f6.png">

MAC Palette Styleguide
![a55ffd4f-14ff-493e-aa13-ba9ca5fb9208](https://user-images.githubusercontent.com/8353860/128076214-cafbae26-cc1e-4bdd-9b26-a94f9e2e4dd5.jpg)

MAC Palette Styleguide Buttons
![55c039ff-37e1-462c-ae2d-2e5f21400afc](https://user-images.githubusercontent.com/8353860/128076248-5c9c2975-cc5b-4361-b0bf-ea8bdd303ba7.jpg)

If we want to keep orange as a highlight, some options with white and black text

AA compliant, standard
<img width="578" alt="Screen Shot 2021-08-04 at 9 25 20 AM" src="https://user-images.githubusercontent.com/8353860/128218397-005b881f-6360-4d4e-99e7-bcdb7f7466e7.png">

AAA compliant, strict
<img width="602" alt="Screen Shot 2021-08-04 at 9 25 29 AM" src="https://user-images.githubusercontent.com/8353860/128218444-c4fc9983-b354-40d8-a172-8c49cb81be02.png">

The orange highlight is also used as the active page in the pagination.
<img width="377" alt="Screen Shot 2021-08-05 at 9 27 43 AM" src="https://user-images.githubusercontent.com/8353860/128386759-f27959b2-3af4-4e40-953f-1dd2b4259fce.png">

**Comment by azle** [comment]
After meeting with POSM it is decided that the active color for the orange and white tab should be replaced with the following

Orange Background: #E88F2A
Blue text: #073C5C

This should also be applied to any active styling using the orange white like the following items
-  the list and grid icon highlight
- active state in pagination

In order to be consistent.
